crates/sui-adapter-transactional-tests/tests/transfer_object/wrap_unwrap.move

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// test impact to versions for wrapping and unwrapping an object
+
+//# init --addresses a=0x0 --accounts A
+
+//# publish
+module a::m {
+    use sui::object;
+    use sui::tx_context::{Self, TxContext};
+
+    struct S has key, store {
+        id: object::UID,
+    }
+
+    struct T has key, store {
+        id: object::UID,
+        s: S,
+    }
+
+    entry fun mint(ctx: &mut TxContext) {
+        sui::transfer::transfer(
+            S { id: object::new(ctx) },
+            tx_context::sender(ctx),
+        );
+    }
+
+    entry fun wrap(s: S, ctx: &mut TxContext) {
+        sui::transfer::transfer(
+            T { id: object::new(ctx), s },
+            tx_context::sender(ctx),
+        );
+    }
+
+    entry fun unwrap(t: T, ctx: &mut TxContext) {
+        let T { id, s } = t;
+        object::delete(id);
+        sui::transfer::transfer(s, tx_context::sender(ctx));
+    }
+}
+
+//# run a::m::mint --sender A
+
+//# view-object 106
+
+//# run a::m::wrap --sender A --args object(106)
```

Comment:
Do we have a version of this test that doesn't put the minted object in storage? (I think we do but just checking)

Reply:
Indeed, it's here: https://github.com/MystenLabs/sui/blob/main/crates/sui-adapter-transactional-tests/tests/child_count/unwrap_never_stored_transfer.move

I think it's worth renaming some of these transactional tests once the dust settles.